privacyidea/static/components/token/views/token.enrolled.hotp.html

```diff
@@ -1,9 +1,12 @@
 <div class="row">
-    <div class="col-sm-6" ng-hide="form['2stepinit'] && enrolledToken.rollout_state !== 'clientwait'">
+    <div ng-hide="form['2stepinit'] && enrolledToken.rollout_state !== 'clientwait'"
+         ng-class="$state.includes('token.wizard') == true && form['2stepinit'] == false ? 'col-sm-12' : 'col-sm-6'">
         <img width="{{ qrCodeWidth }}"
+             ng-class="$state.includes('token.wizard') == true ? 'centered' : 'not-centered'"
              ng-src="{{ enrolledToken.googleurl.img }}"/>
     </div>
-    <div class="col-sm-6" ng-hide="form['2stepinit']">
+    <div ng-hide="form['2stepinit']"
+         ng-class="$state.includes('token.wizard') == true && form['2stepinit'] == false ? 'col-sm-12' : 'col-sm-6'">>
```

Comment:
There is a typo at the end of the line. It must be ">" not ">>".

Reply:
Thank's, changed now.